padertorch/contrib/data/wsj0_mix/create_json.py

```diff
@@ -0,0 +1,273 @@
+import itertools
+
+import functools
+import re
+import tempfile
+from concurrent.futures import ThreadPoolExecutor
+from pathlib import Path
+
+import click
+import numpy as np
+import sh
+from tqdm import tqdm
+
+import paderbox as pb
+
+from ..utils import check_audio_files_exist
+
+"""
+To create the dataset, all files from the wsj database have to be converted to
+wav and stored into one folder. The script 'combine_wsj0.sh' can be used for
+this after changing the database and output paths.
+After this, download the matlab-script
+(wget http://www.merl.com/demos/deep-clustering/create-speaker-mixtures.zip)
+and install voicebox (view readme inside the downloaded zip file for details).
+In the matlab script, change the directories accordingly (wsj0root and output
+paths) and execute the script. This should create some subfolders in the
+specified output folders and a few GB of wav data. By executing this script,
+you can create a JSON-file which can be loaded using a JsonCallbackFetcher.
+
+The transcriptions are taken from the "clean word" section of the wsj dataset.
+For each (original wsj) utterance id there is a transcription in the "orth"
+section. For each merl utterance id there is an annotation entry that maps it
+to the wsj utterance id of the shortest utterance in the mix.
+"""
+
+# Define some keys that are used in the JSON format
+DATASETS = 'datasets'
+SPEAKER_ID = 'speaker_id'
+GENDER = 'gender'
+NUM_SAMPLES = 'num_samples'
+LOG_WEIGHTS = 'log_weights'
+TRANSCRIPTION = 'transcription'
+OBSERVATION = 'observation'
+SPEECH_SOURCE = 'speech_source'
+AUDIO_PATH = 'audio_path'
+
+
+def download_normalize_transcript():
+    """
+    Downloads Kaldis "normalize_transcript.pl" for WSJ to the working directory
+    """
+    normalize_transcript_path = Path('normalize_transcript.pl').resolve()
```

Comment:
Can you use 
```
from appdirs import user_cache_dir
user_cache_dir('paderbox')
```
instead of `cwd`?

Reply:
Or alternatively, write the file next to the json.

Reply:
Done